qiita_db/test/test_analysis.py

```diff
@@ -91,16 +96,55 @@ def _create_analyses_with_samples(self, user='demo@microbio.me',
         """
         user = qdb.user.User(user)
         dflt_analysis = user.default_analysis
+        aid = 4
+        if addtl_processing_cmd:
+            cmd_id = qdb.artifact.Artifact(aid).processing_parameters.command
```

Comment:
note that this is cmd, and the line below is cmd_id

Reply:
I just broke it into two lines to pass flake8. In the next line, cmd_id is reassigned to cmd_id.id. I just didn't want to create an additional variable. It's probably bad form, now that I look at it.